__tests__/offlineUI.js

```diff
@@ -50,17 +49,25 @@ describe('offline UI', async () => {
         };
       },
       pingBotOnLoad: false,
-      setup: () => new Promise(resolve => {
-        const scriptElement = document.createElement('script');
+      setup: () =>
+      Promise.all([
+        window.WebChatTest.loadScript('https://unpkg.com/core-js@2.6.3/client/core.min.js'),
```

Comment:
why are we installing these dependencies inline with the tests? is there another purpose than this?

Reply:
We want to mock DirectLineJS in this test. And DirectLineJS use Observable. Thus, we need to temporarily include `core-js` in the browser to provide Observable for us to mock DLJS.

Reply:
Got it!